docs/querying/sql-translation.md

```diff
@@ -221,6 +226,224 @@ The EXPLAIN PLAN statement returns the following result with plan, resources, an
 ]
 ```
 
+Now consider the following EXPLAIN PLAN for a `REPLACE` query that replaces all the data in the `wikipedia` datasource:
```

Comment:
```suggestion
Example 2: EXPLAIN PLAN for a `REPLACE` query that replaces all the data in the `wikipedia` datasource:
```

Should we do something more formal here, like numbering the examples or something?

Reply:
Good point, done